### PR TITLE
Update icons for shade device class

### PR DESCRIPTION
**Pull request by KablammoNick**
MDI v6.7.96 added icons for roller-shade and roller-shade-closed - this changes shade device class to match, rather than blinds/blinds-open.

## Breaking change

I don't believe this is a breaking change, unless someone is particularly attached to the blinds icons for shades.

## Proposed change

MDI v6.7.96 added icons for roller-shade and roller-shade-closed - this changes shade device class to match, rather than blinds/blinds-open.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]
(will need new screenshot of device class icons, but simple enough if/when change approved)

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by KablammoNick** [comment]
Apologies, renamed the branch to something more useful and it closed this. Hopefully re-opening still works.

File: src/common/entity/cover_icon.ts [hunk 1]
```diff
@@ -86,9 +86,9 @@ export const coverIcon = (state?: string, stateObj?: HassEntity): string => {
         case "closing":
           return mdiArrowDownBox;
         case "closed":
-          return mdiBlinds;
+          return mdiRollerShadeClosed;
```
Comment (bramkragten) [inline src/common/entity/cover_icon.ts:89]: You made this change for both `blind` and `shade` device classes now, was that intentional?

Comment (bramkragten) [reply src/common/entity/cover_icon.ts:89]: I'd suggest we use `blinds-horizontal` and `roller-shade`

Comment (KablammoNick) [reply src/common/entity/cover_icon.ts:89]: No it was not. I didn't realise they were one and the same.
Will have to split the case for blind device class out and leave it as original.

I assume I need to close this pull request and try again with the edits?

Comment (bramkragten) [reply src/common/entity/cover_icon.ts:89]: No, you can continue changing this branch: <https://github.com/KablammoNick/frontend/blob/patch-1/src/common/entity/cover_icon.ts>

Comment (KablammoNick) [reply src/common/entity/cover_icon.ts:101]: Done, also noticed I missed them at the top import.

Thanks for the help.

**Review by piitaya — APPROVED** [review]

**Pull request merged**